packages/mdc-textfield/_variables.scss

```diff
@@ -15,41 +15,27 @@
 //
 
 $mdc-text-field-error-on-light: #d50000;
```

Comment:
We can probably remove the `-on-error` suffix from all of the remaining color variables

Reply:
I removed all the -on-light as well as -light-* variable names.